presets/4.4/tune/karate/karate_brushless_whoop.txt

```diff
@@ -0,0 +1,162 @@
+#$ TITLE: Karate brushless whoop
```

Comment:
"Karate brushless **2S** whoop"?
If this is a 2S. Or it's tested-good on 1S too?

Reply:
its good for 1s brushless too

Reply:
1s is good